This is a fix for:
31538463 - Ghostscript fails to generate output for mgrgray8

Upstream bug:
https://bugs.ghostscript.com/show_bug.cgi?id=702517

This issue is going to be fixed in upstream so the patch should be removed during next uprev.

diff --git a/devices/gdevmgr.c b/devices/gdevmgr.c
index 79c299d..4d61e8a 100644
--- a/devices/gdevmgr.c
+++ b/devices/gdevmgr.c
@@ -218,10 +218,11 @@ mgrN_print_page(gx_device_printer *pdev, gp_file *pstream)
             }
         }
 
-        if ( bdev->mgr_depth != 8 )
+        if ( bdev->mgr_depth != 8 ) {
             data = (byte *)gs_malloc(pdev->memory, mgr_line_size, 1, "mgrN_print_page");
-        if (data == NULL)
-            return_error(gs_error_VMerror);
+            if (data == NULL)
+                return_error(gs_error_VMerror);
+        }
 
         while ( !(code = mgr_next_row(&cur)) )
            {
